wiki/People/pt-br.md

```diff
@@ -0,0 +1,12 @@
+# People
```

Comment:
This wasnt translated?

Reply:
I don't saw anything.

Reply:
No, it wasn't. Just checked right now.

Reply:
@kisuco Can you translate the title too?

Reply:
done.

Reply:
@MegaApplePi